web/html/src/components/input/form-utils.ts

```diff
@@ -63,7 +91,8 @@ export function unflattenModel<T>(flatModel: Record<string, T>): TreeLikeModel<T
   const mergeArrays = (model: any): any => {
     return Object.keys(model).reduce((result, key) => {
       const matcher = key.match(/^([a-zA-Z0-9]*[A-zA-Z])[0-9]+$/);
-      const mergedValue = model[key] instanceof Object ? mergeArrays(model[key]) : model[key];
+      const mergedValue = typeof model[key] === "object" && !Array.isArray(model[key]) ?
```

Comment:
I'm guessing there was a bug here with handling arrays since they're also objects? It would be nice to add one small test case that covers the case that was bugged before but works now. Nice catch. 👍

Reply:
Well, I modified a test for that. See here: https://github.com/uyuni-project/uyuni/pull/3286/files#diff-5eb60c337c8f8aad296dac8a0a9a8d7ee1ec89c89ae9c51b5b64623af90a02dfL74-R87

Reply:
Ah okay, I missed that, sorry. Good fix.